modules/ref_device_module/src/ref_device_impl.cpp

```diff
@@ -244,6 +244,24 @@ void RefDeviceImpl::initProperties()
     objPtr.addProperty(BoolProperty("EnableCANChannel", False));
     objPtr.getOnPropertyValueWrite("EnableCANChannel") +=
         [this](PropertyObjectPtr& obj, PropertyValueEventArgsPtr& args) { enableCANChannel(); };
+
+    auto options = context.getModuleOptions("RefDevice");
+    if (options.getCount() == 0)
+        return;
+
+    if (options.hasKey("numberofchannels"))
```

Comment:
the idea was not to make suffer when user runs program with command line args or env variables
-CModules_NativeStreamingClient_HeartbeatEnabled=true

Reply:
It's a bit odd in the module code, where we are expecting case sensitive property names, but those do not match the names given in the "Options" dictionary - if I were looking at the code it would be unclear to me as to why these don't match.